src/main/java/core/basesyntax/dao/FileReader.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface FileReader {
+    List<FruitTransaction> readFile(String fileName);
```

Comment:
According to the checklist, the `readFile` method should not return `List<FruitTransaction>`. It should return `List<String>`. The parsing logic to convert `String` to `FruitTransaction` should be in a separate service.